bleak/backends/bluezdbus/utils.py

```diff
@@ -6,8 +6,8 @@
 from bleak.backends.bluezdbus import defs
 from bleak.exc import BleakError
 
-_mac_address_regex = re.compile("^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$")
-_hci_device_regex = re.compile("^hci(\d+)$")
+_mac_address_regex = re.compile(r'^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$')
+_hci_device_regex = re.compile(r'^hci(\d+)$')
```

Comment:
I would rather see this changed to 
```
_hci_device_regex = re.compile("^hci(\\d+)$")
```
That woudl remove the warning and keep it as a regular string.

Reply:
fair enough, done